hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: minor
+
+This patch fixes an exception that occurs when using type unions of
+the :pypi:`typing_extensions` ``Literal`` backport on Python 3.6.
```

Comment:
We like to credit our contributors here!  It's a little weird that our release system has *you* write it, but "Thanks to Ben Anhalt for identifying and fixing this bug." would be a lovely signoff.

Also please add yourself to the list in `AUTHORS.rst` 😁